lib/CarriageTheme.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+
+class CarriageTheme {
```

Comment:
This may be too late now, but given this PR and the history PR share this file, maybe what would be best is make the CarriageTheme.dart its own PR (so don't use it anywhere just have it exist) and then have these other two PRs?

Reply:
Sorry about this--it was initially supposed to actually just be minor style changes and then I realized we should just get aligned with the system and forgot to split it into a new PR